documentation/modules/ref-kafka-user.adoc

```diff
@@ -3,30 +3,34 @@
 // assembly-using-the-user-operator.adoc
 
 [id='ref-kafka-user-{context}']
-= Kafka User resource
+= Securing user access to Kafka
```

Comment:
I think that the change of the title is misleading since this only one of the ways. It doesn't cover for example OAuth. So I think this should be still abotu the KafkaUser resource.

Reply:
Good point, though we do mention oauth. Reverted.